Checkout/Checkout/Test/Validation/CVVValidatorTests.swift

```diff
@@ -51,8 +51,9 @@ final class CVVValidatorTests: XCTestCase {
   }
 
   func testValidateCVVIncorrectLengthReturnsCorrectError() {
+    let excludedSchemes = [Card.Scheme.maestro, .unknown]
     Card.Scheme.allCases.forEach { scheme in
-      for length in scheme.cvvLengths where scheme != .unknown {
+      for length in scheme.cvvLengths where !excludedSchemes.contains(scheme) {
```

Comment:
The loop is not mine, its been here since Nov 21 when I think Daven created it :). Could be older or newer as files have been moved recently so history is fuzzy.

From my part in this, the test was covering all 9 cards types. 2 of them have been moved and that leaves 7 cards in the loop. Personally I didn't find it justifiable for the change I did to refactor the whole test and create 7 other unit tests unrelated to my task.

If I was to do it now from scratch I'd do as you mentioned (and as you can see added in the PR), but refactoring this file is extensive, even test naming convention is non standard, so the level of potential improvements to this file is higher.

Reply:
Thanks @alex-ioja-yang-cko makes sense now. I think we should add something in backlog to improve this as tech investment for later. A good candidate to improve our unit tests.